test/OpenTelemetry.Instrumentation.SqlClient.Tests/OpenTelemetry.Instrumentation.SqlClient.Tests.csproj

```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <Description>Unit test project for OpenTelemetry SqlClient instrumentations</Description>
     <!-- OmniSharp/VS Code requires TargetFrameworks to be in descending order for IntelliSense and analysis. -->
-    <TargetFrameworks Condition="$(TARGET_FRAMEWORK) == ''">net6.0</TargetFrameworks>
+    <TargetFrameworks Condition="$(TARGET_FRAMEWORK) == ''">net6.0;net7.0</TargetFrameworks>
```

Comment:
What do we want our practice to be?

In the case of the SqlClient instrumentation, here's what the instrumentation currently targets:

https://github.com/open-telemetry/opentelemetry-dotnet/blob/aba6eafb8f62614f4035e280bef77d07c21c569a/src/OpenTelemetry.Instrumentation.SqlClient/OpenTelemetry.Instrumentation.SqlClient.csproj#L5

So both net6.0 and net7.0 test runs will be testing the same instrumentation code - i.e., the netstandard2.0 build. Additional targets to the test project will extend the CI time by a little bit. Is there a benefit? Is it that the test will run on different versions of the runtime?

Reply:
if we run on net6,7, etc, we need to modify the test to use corresponding version of sqlclient. (I dnt know if sqlclient is version aligned with .net cycles..)

Reply:
This is what we use https://github.com/open-telemetry/opentelemetry-dotnet/blob/0be837f89592576ced5cd7323d15a69999bb179b/test/OpenTelemetry.Instrumentation.SqlClient.Tests/OpenTelemetry.Instrumentation.SqlClient.Tests.csproj#L18. I don't think it varies with runtime. we could just keep `net6.0` target.

Reply:
In that case, I think it's best to not add the `net7.0` targets to most of the other test projects you've added it to. At least not until we have a good reason to.

Reply:
Done